radix-cluster-cleanup/main.go

```diff
@@ -2,6 +2,7 @@ package main
 
 import (
 	"github.com/equinor/radix-cluster-cleanup/cmd"
+
 	// Force loading of needed authentication library
 	_ "k8s.io/client-go/plugin/pkg/client/auth/azure"
```

Comment:
You can remove k8s.io/client-go/plugin/pkg/client/auth/azure

It is deprecated. Ref https://github.com/kubernetes/client-go/blob/master/plugin/pkg/client/auth/azure/azure_stub.go